### DISCUSSION TITLE
Date not selectable in form field

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#13731] [Help] [answered]
Hi all, I create a Budibase app and connected with MYSQL database with two tables. In the tables there are two auto dates 'created_dt' and 'modified_dt'. These fields are automatically filled or updated when a row is inserted or updated. 

I want to view these fields in a form. But when I use Fromblock or something similar the dates are not selectable. Is there a way to make these dates visible in a form?

Thank you all for all the help.

**Comment by mikesealey** [comment, marked as answer]
Hi @JeroenP1974 I've been looking into this for you.

You're right, those values aren't available to select in forms, as they're auto-columns in MySQL. Their values, however, can still be read.

As a rough demonstration, I've created a MySQL table with `id`, `data`, `created_dt`, `modified_dt`, where `data` is a string, and `created_dt` and `modified_dt` are both automatically generated.

I then created a page using the budibase template Table with details panel, selected the table, and the access level...
![image](https://github.com/Budibase/budibase/assets/39269380/98cccc42-91f8-4cf6-a84c-e5884288a90d)

When clicking a row in the table, the row-id is set in state and a side panel opens. Inside the side panel, I added a paragraph underneath the form. When setting the value for the paragraph, the bindings menu then has access to `Edit row form block - row`, which contains every key and value from that row .
![image](https://github.com/Budibase/budibase/assets/39269380/1fe9ab01-0ef8-46be-a18a-1bf4fecb40c9).

![image](https://github.com/Budibase/budibase/assets/39269380/7ef281c2-7dfa-4154-8e95-7144e2e4d4fe)

If you're taking a different approach, you may need to set the row in state, and fetch it's details that way.

I hope this helps. If you haven't already, consider joining the [Budibase Discord](https://discord.gg/ZepTmGbtfF).
